Cura 4.3 slicing fails; missing setting 'machine_heated_build_volume'

**Describe the bug**
Slicing fails on latest Cura (4.3). When trying to slice any model the slicing progress bar will
get stuck at about ~20%. I tried various printer presets and the problem persists. From looking
at the logs it seems like that CuraEngine is unable to get a value for 'machine_heated_build_volume', which was [newly introduced in 4.3](https://github.com/Ultimaker/Cura/pull/6202)

**To Reproduce**
Steps to reproduce the behavior:
1. Choose any printer preset
2. Load any model
3. Press Slice

**Cura log**

relevant parts to the problem seem to be
```
2019-10-15 15:47:44,002 - DEBUG - [Thread-14] UM.Backend.Backend._backendLog [101]: [Backend] [ERROR] Trying to retrieve setting with no value given: 'machine_heated_build_volume'
2019-10-15 15:47:44,192 - DEBUG - [MainThread] UM.Backend.Backend._logSocketState [179]: Socket state changed to Closing
2019-10-15 15:47:44,193 - DEBUG - [MainThread] UM.Backend.Backend._onSocketError [201]: Socket debug: Arcus Error (13): Closing socket because other side requested close.
2019-10-15 15:47:44,193 - DEBUG - [MainThread] UM.Backend.Backend._logSocketState [181]: Socket state changed to Closed
2019-10-15 15:47:44,201 - DEBUG - [MainThread] CuraEngineBackend.CuraEngineBackend._onBackendQuit [806]: Backend quit with return code 2. Resetting process and socket.
```
**aditional information**

it seems like `share/resources/definitions/ultimaker_s5.def.json`, from the nix package, indeed is missing the new setting. For comparison the [upstream definition](https://github.com/Ultimaker/Cura/blob/f947269cf88b9d716c29195a1a0ceb442b093cee/resources/definitions/ultimaker_s5.def.json#L45-L48
).

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.5, NixOS, 20.03pre196725.94500c93dc2 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre196725.94500c93dc2"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```
Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: [cura, curaengine]
# a list of nixos modules affected by the problem
module: []
```

--- a/pkgs/applications/misc/cura/default.nix
+++ b/pkgs/applications/misc/cura/default.nix
@@ -8,7 +8,7 @@ mkDerivation rec {
     owner = "Ultimaker";
     repo = "Cura";
     rev = version;
-    sha256 = "1qnai8vmgy5lx3lapw96j41i8mw9p6r99i3qzs709l9yzrix6l86";
+    sha256 = "1wf60qr0wqsci5skp55qr8h56s3x5s2icxbn58ia9s4a5hhvnsmf";
   };
 
   materials = fetchFromGitHub {

